.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "helio"]
 	path = helio
 	url = https://github.com/romange/helio.git
+[submodule "wasmtime"]
+	path = wasmtime
```

Comment:
We should really use `add_third_party`. I couldn't make it work and I didn't want to loose time since this is just a prototype merely to show how we embed wasmtime, how we export API functions to sdk users, and how we consume wasm binaries in dragonfly